anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -3,6 +3,7 @@
 from itertools import product
 
 import numpy as np
+from anisoap_rust_lib import compute_moments
```

Comment:
Do we instead want to make this detect whether rust version is installed, and only use compute_inefficient if rust is not installed? I'm hesitant to add another function argument.

Reply:
I think the rust version should just be automatically installed, and we should require the rust compiler and a compiler tool chain as a prerequisite so the code is built from source. Once we put the code on pypi, we can make prebuilt wheels and loosen the requirements for compilers and build tools as the distributed versions will be prebuilt

Reply:
My thought is that the rust version should always be used by the end user. For the developer, we will retain the option to use the original python version in case something with the rust version goes wrong.